test/python/common.py

```diff
@@ -14,39 +16,36 @@
 class NgraphTest(object):
 
     def with_ngraph(self, l, config=tf.ConfigProto()):
-        ngraph_tf_disable = os.environ.pop('NGRAPH_TF_DISABLE', None)
         ngraph_tf_disable_deassign_clusters = os.environ.pop(
             'NGRAPH_TF_DISABLE_DEASSIGN_CLUSTERS', None)
 
+        os.environ['NGRAPH_TF_LOG_PLACEMENT'] = '1'
         os.environ['NGRAPH_TF_DISABLE_DEASSIGN_CLUSTERS'] = '1'
-
+        ngraph.enable()
         with tf.Session(config=config) as sess:
             retval = l(sess)
 
         os.environ.pop('NGRAPH_TF_DISABLE_DEASSIGN_CLUSTERS', None)
 
-        if ngraph_tf_disable is not None:
-            os.environ['NGRAPH_TF_DISABLE'] = ngraph_tf_disable
         if ngraph_tf_disable_deassign_clusters is not None:
             os.environ['NGRAPH_TF_DISABLE_DEASSIGN_CLUSTERS'] = \
                 ngraph_tf_disable_deassign_clusters
 
         return retval
 
     def without_ngraph(self, l, config=tf.ConfigProto()):
-        ngraph_tf_disable = os.environ.pop('NGRAPH_TF_DISABLE', None)
         ngraph_tf_disable_deassign_clusters = os.environ.pop(
             'NGRAPH_TF_DISABLE_DEASSIGN_CLUSTERS', None)
 
+        os.environ['NGRAPH_TF_LOG_PLACEMENT'] = '1'
```

Comment:
Why do we have the Log Placement set?

Reply:
Ah, forgot to remove that debugging line, since purest swallows up all prints without `-s`